source/Core/Src/gui.cpp

```diff
@@ -894,6 +902,30 @@ static void settings_displayPowerPulse(void) {
     OLED::print(OffString);
   }
 }
+
+static bool settings_setAnimationLoop(void) {
+  systemSettings.animationLoop++;
+  systemSettings.animationLoop = systemSettings.animationLoop % 2;
+  return systemSettings.animationLoop == 1;
+}
+
+static void settings_displayAnimationLoop(void) {
+  printShortDescription(29, 7);
+  OLED::print(systemSettings.animationLoop ? SymbolPlus : SymbolMinus);
```

Comment:
Should we use the checkbox or True/False instead of plus minus?

Reply:
Forgot about checkbox